test/DynamoCoreTests/LibraryTests.cs

```diff
@@ -298,19 +298,22 @@ public void TestBuiltInFunctionRecognition()
         [Category("UnitTests")]
         public void GetAllFunctionDescriptorsTest()
         {
-            IEnumerable<FunctionDescriptor> descriptors;
+            const string libraryPath = "FFITarget.dll";
+            if (!libraryServices.IsLibraryLoaded(libraryPath))
+            {
+                libraryServices.ImportLibrary(libraryPath);
+                Assert.IsTrue(LibraryLoaded);
+            }
             
-            descriptors = libraryServices.GetAllFunctionDescriptors("DSCore.Math.Round");
-            Assert.AreEqual(2, descriptors.Count()); // Math.Round has two overloads
+            string name = "FFITarget.TestOverloadConstructor.TestOverloadConstructor";
 
-            descriptors = libraryServices.GetAllFunctionDescriptors("DSCore.Color.Add");
-            Assert.AreEqual(1, descriptors.Count()); // Color.Add has only one overload
-
-            descriptors = libraryServices.GetAllFunctionDescriptors("Invalid node name");
-            Assert.IsNull(descriptors);
+            // Get the function groups that are named FFITarget.TestOverloadConstructor.TestOverloadConstructor
+            var descriptors = libraryServices.GetFunctionGroups(libraryPath)
```

Comment:
It looks like we are testing for both `GetFunctionGroups` as well as `GetAllFunctionDescriptors` in this unit test. Can't we simply test for the result from `GetAllFunctionDescriptors` against the constant number? That number should not change since they belong to `FFITarget`.